### ISSUE TITLE
v12(containers): timezone in pf.conf prevent pfconfig to start correctly

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#6959]
**Describe the bug**
If you have a timezone defined in `pf.conf`, it prevents `packetfence-config` to start correctly.

**To Reproduce**
1. Add `timezone=UTC` in `pf.conf` under `general` section
2. Do a configreload hard
3. Restart PacketFence config

=> You got following logs in `journalctl`:

```
Apr 07 06:02:48 pfdeb11localdev pfconfig-docker-wrapper[17159]: Preloading config::Pf()
Apr 07 06:02:48 pfdeb11localdev pfconfig-docker-wrapper[17159]: WARNING: The timezone is being changed from Etc/UTC to UTC on the system. It is advised to reboot the server so that all services start with the correct timezone.
Apr 07 06:02:48 pfdeb11localdev pfconfig-docker-wrapper[17159]: System has not been booted with systemd as init system (PID 1). Can't operate.
Apr 07 06:02:48 pfdeb11localdev pfconfig-docker-wrapper[17159]: Failed to connect to bus: Host is down
Apr 07 06:02:49 pfdeb11localdev pfconfig-docker-wrapper[17159]: Unable to set timezone on the system
Apr 07 06:02:49 pfdeb11localdev pfconfig-docker-wrapper[17159]: pfconfig(8) FATAL: [mac:[undef]] Unable to set timezone on the system
Apr 07 06:02:49 pfdeb11localdev pfconfig-docker-wrapper[17159]:  (pfconfig::namespaces::config::Pf::set_timezone)
Apr 07 06:02:49 pfdeb11localdev systemd[1]: packetfence-config.service: Main process exited, code=exited, status=25/n/a
Apr 07 06:02:49 pfdeb11localdev systemd[1]: packetfence-config.service: Failed with result 'exit-code'.
Apr 07 06:02:49 pfdeb11localdev systemd[1]: packetfence-config.service: Scheduled restart job, restart counter is at 1.
Apr 07 06:02:49 pfdeb11localdev systemd[1]: Stopped PacketFence Config Service.
```

**Expected behavior**
`pfconfig` should start

**Additional context**
We need to adjust code of [set_timezone function](https://github.com/inverse-inc/packetfence/blob/257c35dfa6d7023cde1927953c8e2fcae5f615e6/lib/pfconfig/namespaces/config/Pf.pm#L198-L208) because I think `pfconfig` is trying to change timezone in its own container.

**Comment by julsemaan** [comment]
There are 2 things that come out of this issue for me:
- We need to make the timezone setting happen outside of pfconfig for on-prem (start-pre or something like that)
- For cloudnac/k8s, we need to disable this mechanism and handle it differently (ex: forcing a UTC timezone if it's empty)

**Comment by nqb** [comment]
Still an issue on `feature/pfconfig-container` branch. It prevents restart of `pfqueue` service at end of configurator test suite because `pfconfig` is restarting in a loop. 

Removing `timezone=UTC` in `pf.conf` + `configreload hard` + restart of `packetfence-config` is a workaround.

**Comment by julsemaan** [comment]
Just replicated this in a recent build. pfperl-api is able to interract with systemd given the appropriate mounts and docker options so giving them to pfconfig may get this resolved. I'll try it out and push a fix when I have one

**Comment by julsemaan** [comment]
fixed in 9249ac9197fdd5fa42bcdd717968942c443f0302
